### ISSUE TITLE
Ray doesn't show file that executed ray helper

### ISSUE THREAD

**Issue by GraxMonzo** [spatie/laravel-ray#3]
Windows 10, IIS

Laravel v8.21.0

Controller:
```
class MyController extends Controller
{
    ...
    public function index()
    {
        ray()->showQueries();

        ray('Hello world');

        return $this->repository->allCarWashes();
    }
    ...
}
```

![image](https://user-images.githubusercontent.com/11737104/103913333-9f6f0580-5132-11eb-9d4c-f955d3f23ee0.png)

**Comment by GraxMonzo** [comment]
Changing 'ray/src/helpers.php' to  'ray\src\helpers.php' helps on Windows
https://github.com/spatie/laravel-ray/blob/8eb9f9ad37a64f8327f032baca08c0fed78e558e/src/OriginFactory.php#L52

**Comment by GraxMonzo** [comment]
Resolved in PR #20
